docs/sources/flow/tasks/metamonitoring.md

```diff
@@ -0,0 +1,166 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/tasks/setup-metamonitoring/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/tasks/setup-metamonitoring/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/tasks/setup-metamonitoring/
+- /docs/grafana-cloud/send-data/agent/flow/tasks/setup-metamonitoring/
+# Previous page aliases for backwards compatibility:
+- /docs/grafana-cloud/agent/flow/getting-started/setup-metamonitoring/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/getting-started/setup-metamonitoring/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/getting-started/setup-metamonitoring/
+- /docs/grafana-cloud/send-data/agent/flow/getting-started/setup-metamonitoring/
+- ../getting-started/setup-metamonitoring/ # /docs/agent/latest/flow/getting-started/setup-metamonitoring/
```

Comment:
We can remove ALL of these aliases. These are only required if we move/rename a topic that has already been published. This is a new topic... so no aliases required.

Reply:
Thanks, TIL!